recipes/rg

```diff
@@ -0,0 +1,4 @@
+(rg
+ :fetcher github
+ :repo "dajva/rg.el"
+ :branch "melpa")
```

Comment:
Agreed. We won't pull off a branch like that, because you might forget to update it. MELPA wants to build packages which automatically track your ongoing development, and you can apply version tags to control what "release" packages get built in MELPA Stable.